### PR TITLE
feat: wallet-server send chapi request

### LINKED ISSUE #652: CHAPI Bridge : wallet-server send chapi request

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- Closes #652
Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
Is this being renamed to be more generic and being moved to adapters in a followup?

**Review by llorllale — APPROVED** [review]

File: pkg/restapi/wallet/chapibridge/operations.go [hunk 1]
```diff
@@ -215,9 +239,70 @@ func (o *Operation) RequestApplicationProfile(rw http.ResponseWriter, req *http.
...
+		"data":  request.Body.Request,
+	})
+	if err != nil {
+		logutil.LogError(logger, commandName, SendCHAPIRequest, err.Error())
+		common.WriteErrorResponsef(rw, logger, http.StatusInternalServerError, err.Error())
+
+		return
+	}
+
+	responseBytes, err := o.messenger.Send(msgBytes,
```
Comment (llorllale) [inline pkg/restapi/wallet/chapibridge/operations.go:281]: If I understand correctly, this message of type `https://trustbloc.dev/chapi/1.0/request` will be received by the "Mobile Mediator".

Why isn't a `forward` message used to transport this chapiRequest payload to the mediator?

**Review by rolsonquadras — APPROVED** [review]

**Comment by rolsonquadras** [comment]
@sudeshrshetty @llorllale @troyronda - sorry, I missed to notice the comments before I merged this.

**Comment by sudeshrshetty** [comment]
> Is this being renamed to be more generic and being moved to adapters in a followup?

We will be using new path names once we move this to `edge-adpapters`

File: pkg/restapi/wallet/chapibridge/operations.go (hunk 1 above)
Comment (llorllale) [reply pkg/restapi/wallet/chapibridge/operations.go:281]: Right, thanks.

**Pull request merged**